packages/web-api/package.json

```
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "@slack/logger": "^4.0.0",
-    "@slack/types": "^2.9.0",
+    "@slack/types": "2.16.0-aiAppsBeta.3",
```

suggestion: We'll need to revert this version
```suggestion
    "@slack/types": "2.16.0",
```

---

@mwbrooks Excellent catch, as always - Thank you 👾 ✨

---

@mwbrooks We reverted this in https://github.com/slackapi/node-slack-sdk/pull/2399/commits/c41adc99e4351295c99562cb2f5f0feb9296b302 to use the prerelease changes in tests, but we'll revert this before the next release of `@slack/web-api` 🫡